go.mod

```diff
@@ -31,7 +31,7 @@ require (
 	github.com/ncw/swift v1.0.47
 	github.com/opencontainers/go-digest v1.0.0-rc1
 	github.com/opencontainers/image-spec v1.0.1
-	github.com/satori/go.uuid v1.2.0 // indirect
+	github.com/satori/go.uuid v0.0.0-20181028125025-b2ce2384e17b // indirect
```

Comment:
Not sure if `go.mod` preserves comments (I think it does); if so, it may be useful to add a comment describing why we used this commit, something along the lines of

```
// v1.2.0-12-gb2ce238, includes 75cca531 (WS-2018-0594), which is not in a tagged release
```

Reply:
Thanks, I used your comment as-is.